api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirCohortListServiceImpl.java

```diff
@@ -32,8 +33,13 @@
 	private ListTranslator<Cohort> cohortListTranslator;
 	
 	@Override
-	@Transactional(readOnly = true)
-	public ListResource getListByUuid(String uuid) {
-		return cohortListTranslator.toFhirResource(dao.getListByUuid(uuid));
+	public FhirListDao<Cohort> getDao() {
+		return dao;
 	}
+	
+	@Override
+	protected OpenmrsFhirTranslator<Cohort, ListResource> getTranslator() {
+		return cohortListTranslator;
+	}
```

Comment:
Preferably, annotated the class with  `@Getter(AccessLevel.PROTECTED)`

Reply:
You will have to change `ListTranslator<Cohort> cohortListTranslator;` to `ListTranslator<Cohort> translator;`